.github/actions/kind/action.yml

```diff
@@ -25,10 +32,20 @@ runs:
       shell: bash
       run: |
         kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.4/deploy/static/provider/kind/deploy.yaml
+        # waiting for resources to be created
+        kubectl rollout status deployment ingress-nginx-controller --namespace ingress-nginx
```

Comment:
After my changes the line below was failing on non existing pod, I guess it was caused by KinD not being fully ready yet (maybe because now we don't wait for KinD to be running?). This line makes sure that Ingress Deployment is ready